awx/main/models/notifications.py

```diff
@@ -484,14 +485,27 @@ def build_notification_message(self, nt, status):
         if msg_template:
             try:
                 msg = env.from_string(msg_template).render(**context)
-            except (TemplateSyntaxError, UndefinedError, SecurityError):
-                msg = ''
+            except (TemplateSyntaxError, UndefinedError, SecurityError) as e:
+                msg = e  # __str__
+                msg += '\n'.join(traceback.format_exception(None, e, e.__traceback__))
```

Comment:
This is not going to work in the way that you expect.

```
In [1]: try:
   ...:     raise Exception("foo")
   ...: except Exception as e:
   ...:     msg = e
   ...:     msg += "bar"
   ...: 
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
Input In [1], in <cell line: 1>()
      1 try:
----> 2     raise Exception("foo")
      3 except Exception as e:

Exception: foo

During handling of the above exception, another exception occurred:

TypeError                                 Traceback (most recent call last)
Input In [1], in <cell line: 1>()
      3 except Exception as e:
      4     msg = e
----> 5     msg += "bar"

TypeError: unsupported operand type(s) for +=: 'Exception' and 'str'
```

Reply:
Also, using `+=` to build up strings is an anti-pattern in Python.  I'd instead suggest something like

```
msg = '\n'.join([
    str(e),
    ...whatever...
])
```